tuf/api/serialization/json.py

```diff
@@ -47,19 +49,35 @@ class JSONSerializer(MetadataSerializer):
     Attributes:
         compact: A boolean indicating if the JSON bytes generated in
                 'serialize' should be compact by excluding whitespace.
+        validate: Whether to validate when serializing. It's used to prevent
+            common programming mistakes when changing the metadata objects.
```

Comment:
I think this is unnecessarily vague. What about:

```suggestion
        validate: Check that the metadata object can be deserialized again
            without change of contents. This validation might slow down
            serialization significantly.
```

Reply:
I agree with you, but also I want to mention that this helps find common mistakes.
So, I made it: 
```
Check that the metadata object can be deserialized again
	            without change of contents and thus find common mistakes. This
                validation might slow down serialization significantly.
```